_includes/american-copyright-section.html

```diff
@@ -0,0 +1,6 @@
+<section class="c-american-copyright">
+  <div class="c-american-copyright__container">
+    <img src="../assets/american-ailrines-logo.svg" alt="American Airlines logo" class="c-american-copyright__image">
+    <p class="c-american-copyright__text">A wholly owned subsidiary of American Airlines</p>
```

Comment:
Make American Airlines stay together by adding a non-breaking space: `American&nbsp;Airlines`.

I also wonder if at small screen it would help to have this text centered.

![Screen Shot 2019-08-20 at 11 50 39 am](https://user-images.githubusercontent.com/1128391/63363049-e3484b80-c340-11e9-9323-1ce3d9d14488.png)
This is at around 420px wide

Reply:
true. 😄 fixed!